src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -119,6 +122,7 @@ public RESTZaak partialUpdateZaak(@PathParam("uuid") final UUID zaakUUID, final
         final Zaak zaak = new Zaak();
         zaak.setToelichting(restZaak.toelichting);
         zaak.setOmschrijving(restZaak.omschrijving);
+        zaak.setStartdatum(restZaak.startdatum);
```

Comment:
Eigenlijk zou het aanmaken van de zaakl en toekennen van de drie waarden via een, eventuele nieuwe, methode van de zaakConverter moeten. zaakConverter.convertToZaakPatch() of zo

Reply:
Ik heb er een convertToPatch methode van gemaakt, omdat het al impliciet om een zaak gaat